### PR TITLE
Removes standard resources from config tree

### PR DESCRIPTION
**Pull request by Cyberboss**
The game should be able to run just fine without the entire /config directory

### CODE REVIEW

File: code/controllers/subsystem/title.dm [hunk 1]
```diff
@@ -36,11 +36,14 @@ SUBSYSTEM_DEF(title)
 				break
 
 		file_path = "config/title_screens/images/[pick(title_screens)]"
-		
-		icon = new(fcopy_rsc(file_path))
+	
+	if(!file_path)
+		file_path = "icons/default.dmi"
```
Comment (Cyberboss) [inline code/controllers/subsystem/title.dm]: Someone actually wrote out default.dmi wtf

**Comment by optimumtact** [comment]
I disagree with this, they get the config folder for free when downloading via github, why would we want to support running without it.

**Comment by Cyberboss** [comment]
I don't want to encourage the idea of the config as a resource repository
because it means if we change/add something people need to manually
download/copy stuff from GitHub if their config isn't part of VCS (i.e.
tgs).

**Comment by Cyberboss** [comment]
Also we don't need to specifically license stuff we put in this folder
anymore

**Comment by optimumtact** [comment]
okay fair enough

**Pull request merged**